composer.json

```diff
@@ -8,7 +8,7 @@
         "php": ">=7.4 <8.3",
         "ext-json": "*",
         "phpstan/phpstan": "^1.10.6",
-        "phpstan/phpstan-nette": "^1.2.6",
+        "phpstan/phpstan-nette": "1.2.6",
```

Comment:
freezed version of phpstan-nette to working one

Reply:
It is discussed here https://github.com/efabrica-team/phpstan-latte/pull/326#issuecomment-1505761868
phpstan-nette 1.2.6 was last (or only version) where our tests for Multiplier passed. But we will probably ignore those errors. @MartinMystikJonas can you prepare PR and unfreeze phpstan-nette?

Reply:
@spaze version 0.11.0 was released with updated phpstan-nette extension

Reply:
Thanks!